### PR TITLE
Adding sponsor logos

### PR DESCRIPTION
**Pull request by jean**

### CODE REVIEW

File: site/conf.py [hunk 1]
```diff
@@ -1062,16 +1061,79 @@
...
+      <!--
+      <div class="col col-sm-4">
+        <div style="background-color:#FFFFFF;border:4px solid #DFDFDF;width:252px;height:108px;display:flex;justify-content:center;align-items:center;margin-left:auto;margin-right:auto;padding:10px;">
+          <a href="https://www.cloudflare.com/" alt="Cloudflare" style="width:100%;height:100%;background-image:url(/cf-logo-h-rev.svg);background-size:contain;background-repeat:no-repeat;background-position:center;">
+              </a>
+        </div>
+      </div>
+      -->
+    </div>
+    <h3>Individual Sponsors</h2> <!-- TODO: translate -->
```
Comment (djay) [inline site/conf.py]: don't show these on all pages. Just on the sponsors page.

**Review by djay — DISMISSED** [review]

File: site/conf.py [hunk 2]
```diff
@@ -1100,16 +1141,62 @@
...
+  </div>
+</div>
 <div class="navbar-inverse row" style="
     color: white;
 ">
   <div class="container">
-    <div class="col-md-3">
+    <div class="col-md-4">
         <h3>About</h3>
         PyCon Thailand คือ งานสัมนาเกี่ยวกับภาษาที่ใช้เขียนโปรแกรม (Python programming language)เป็นสังคมที่ขับเคลื่อนโดยสมาชิกของกลุ่ม
```
Comment (vtno) [inline site/conf.py]: PyCon Thailand คือ งานสัมนาเกี่ยวกับภาษา Python เป็นงานที่ถูกขับเคลื่อนโดยกลุ่มของนักพัฒนาภาษา Python ในกรุงเทพ

**Comment by djay** [comment]
@jean  ACommerce haven't paid yet I believe? 
I think Ben put them in the spreadsheet by mistake.

I think on sponsor page logos should go at top with some kind of lead in like we want to thank these sponsors for making the event a success and subsidizing tickets for others etc.

Also, can we get a gap between sponsors on and footer on each page?

**Comment by jean** [comment]
@djay please check again?

@tonytonybell @vtno can search for "TODO: translate" to find things that need translation.

**Pull request merged**